third_party/forked/celopenapi/model/schemas.go

```diff
@@ -0,0 +1,398 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"k8s.io/kube-openapi/pkg/validation/spec"
```

Comment:
Any thoughts on a possibly moving toward a `Schema` interface, or factory of sorts so we could swap the internals as needed in the future. I'm asking more from the standpoint of creating a standalone CEL library for this than I am about what to do right this moment.